nasa/earthdata-search
# EDSC-3885: Improving display of variable panel navigation

# Overview

### What is the feature?

Improves the display of the "Browse by" navigation on the variable results panel

### What is the Solution?

Fixed some of the colors and alignment. Removed the disabled class in favor of a css class to denote when and item is active.

<img width="853" alt="image" src="https://github.com/nasa/earthdata-search/assets/7462238/7f164d34-5016-42e6-bd07-80b3c5cf4ba6">

### What areas of the application does this impact?

None

# Testing

### Reproduction steps

- **Environment for testing:** SIT
- **Collection to test with:** C1200277763-E2E_18_4

1. Add a granule to your project
2. Move to the project page
3. Select the OPENDaP access method
4. Open the variables panel
5. Confirm the navigation looks and works as intended

### Attachments

Please include relevant screenshots or files that would be helpful in reviewing and verifying this change.

# Checklist
